funky_time.py

```diff
@@ -16,18 +16,26 @@ def years_ago(years: int, from_date: datetime = None):
         return from_date.replace(month=2, day=28, year=from_date.year - years)
 
 
-# using years_ago, is this person 18 years old, or older, right now?
-def is_18(birth_date: datetime = None):
+# is the `birth_date` `greater` than `age` now?
+def is_age(age: int, greater: bool, birth_date: datetime = None):
```

Comment:
instead of passing a bool to change the functionality of this function, I think it would be clearer to have two separate functions. Maybe something like `is_older_than(age, birth_date)` and `is_younger_than(age, birth_date)`

Reply:
Fair cop! That's done